.github/workflows/benchmarks.yml

```diff
@@ -106,7 +106,10 @@ jobs:
           BENCHMARKS_ARCHIVE: benchmarks.tar
           BENCHMARKS_DIR: ${{ steps.build.outputs.benchmarks-dir }}
         run: |
-          tar --exclude '*.tflite' --exclude '*.mlir' -czf \
+          tar --exclude '*.tflite' --exclude '*.mlir' \
+            --exclude "${BENCHMARKS_DIR}/benchmark_suites/models" \
```

Comment:
Can you add a comment about these exclusions?

Reply:
Done.